config/superglue-bert.conf

```diff
@@ -0,0 +1,89 @@
+// Config settings used for SuperGLUE BERT baseline experiments. 
+
+// This imports the defaults, which can be overridden below.
+include "defaults.conf"
+exp_name = "bert-large-cased"
+
+// Data and preprocessing settings
+max_seq_len = 256 // Mainly needed for MultiRC, to avoid over-truncating
+                  // But not 512 as that is really hard to fit in memory.
+tokenizer = "bert-large-cased"
+
+// Model settings
+bert_model_name = "bert-large-cased"
+bert_embeddings_mode = "top"
+bert_fine_tune = 1
+elmo = 0
+elmo_chars_only = 0 
+pair_attn = 0 // shouldn't be needed but JIC
+s2s = {
+    attention = none
+}
+sent_enc = "none"
+sep_embs_for_skip = 1
+classifier = log_reg // following BERT paper
+transfer_paradigm = finetune // finetune entire BERT model
+
+// Training settings
+dropout = 0.1 // following BERT paper
+optimizer = bert_adam
+batch_size = 4
+max_epochs = 10
+lr = .00001
+min_lr = .0000001
+lr_patience = 4
+patience = 20
+max_vals = 10000
+
+// Control-flow stuff
+do_pretrain = 1
+do_target_task_training = 1
+do_full_eval = 1
+write_preds = val,test  // Somewhat slow, but should be useful for analysis.
+write_strict_glue_format = 1
+
+// These override the task-specific configs set in defaults.conf
+cola = {}
+cola_classifier_dropout = 0.1
+cola_val_interval = 100
```

Comment:
For the smallest SuperGLUE tasks, this is pretty high—potentially over an epoch, even with our tiny batch size.

Reply:
It's also a bit odd that this doesn't match the BOW setup due to the batch size diff, though not fatal.

Reply:
Oh these settings aren't actually used, I forget why I copied them here.
I was planning on adding the validation intervals to the actual script, because it depends directly on batch size and I wanted to make that obvious.

Reply:
The BOW point is fair though...